### DISCUSSION TITLE
Using _sum on findmany function

### DISCUSSION THREAD

**Discussion by volkankorkmaz** [prisma/prisma#8590] [Q&A]
Hi everybody,

I have 2 tables as the below:

- Invoice
- InvoiceItem ( invoiceId, amount) 

I want to get the sum(amount) from InvoiceItem! Can you help me? Thx

Code:
```ts
list = ctx.prisma.invoice.findMany({
              skip: take * skip,
              take,
              where: {
                AND: [{ storeId }, { isDelete: false }, { supplierId }],
              },
              include: {                
                invoiceItems: {
                  select: {
                    _sum: {
                      amount: true,
                      as: 'totalAmount',
                    },
                    groupBy: {
                      invoice: true,
                    },
                  },
                },             
                location: true,
              },
              orderBy: [{ comment: sortType }],
            })
```

Query:
```graphql
query getInvoicesByFilter($input: byFilterInputPurchaseInvoice) {
  getInvoicesByFilter(input: $input) {
    list {
      id
      invoiceDate
      invoiceNumber
      TotalAmount
      comment          
    }
    totalCount
    responseCode
    responseMessage
  }
}
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
